apps/web/context/Auth/context.tsx

```diff
@@ -1,50 +1,39 @@
+'use client';
+
 import { apiHandler } from '@app/config';
 import { Child, User } from '@app/types';
+import { useUser } from '@app/hooks/api/Auth';
+import React, { useMemo, createContext, useContext } from 'react';
 import { useRouter } from 'next/router';
-import React, {
-    useMemo,
-    useState,
-    createContext,
-    useEffect,
-    Dispatch,
-    SetStateAction,
-} from 'react';
 
 interface IAuthTypes {
     logOut: () => Promise<void>;
     user: User | null;
-    login: () => Promise<void>;
-    setUser: Dispatch<SetStateAction<User | null>>;
+    isLoading: boolean;
 }
 export const AuthCtx = createContext({} as IAuthTypes);
 
 export const AuthContext = ({ children }: Child) => {
-    const router = useRouter();
-    const [user, setUser] = useState<User | null>(null);
-    const data = useMemo(
-        () => ({
-            logOut: async () => {
-                setUser(null);
-                apiHandler.post('/auth/logout');
-                router.push('/');
-            },
-            login: async () => {
-                try {
-                    const { data } = await apiHandler.get('/user');
-                    setUser(data);
-                } catch (error) {
-                    console.log(error);
-                }
-            },
-            user,
-            setUser,
-        }),
-        [user],
-    );
+    const { data, isLoading } = useUser();
+    if (typeof window !== undefined) {
+        // localStorage.setItem('access_token', data.access);
+        // localStorage.setItem('refresh_token', data.refresh);
+    }
+    const logOut = async () => {
+        await apiHandler.post('/auth/logOut');
```

Comment:
cant access localstorage it says window not defined need some help here

Reply:
Same here. Why are you storing access Token and Refresh Token

Reply:
so i dont need to send token on calling refresh ? ok got it